app/services/course/assessment/question/programming/language_package_service.rb

```diff
@@ -60,14 +68,37 @@ def extract_meta(attachment, template_files) # rubocop:disable UnusedMethodArgum
     raise NotImplementedError, 'You must implement this'
   end
 
+  # Retrieves the absolute path of the file specified
+  #
+  # @param [String] curr_dir The current directory of the specific language package service
+  # @param [String] filename The filename of the file to get the path of
+  def self.get_file_path(curr_dir, filename)
+    File.join(File.expand_path(File.dirname(__FILE__)), curr_dir, filename).freeze
```

Comment:
Doesn't work, i've refactored it as a private method in each `*_package_service` instead, and moved the variables into the function (since they are only being used once).